deployment/helm/node-feature-discovery/values.yaml

```
@@ -492,20 +504,19 @@ topologyUpdater:
       drop: [ "ALL" ]
     readOnlyRootFilesystem: true
     runAsUser: 0
-
-  # livenessProbe: {}
-    ## NOTE: Currently not configurable, defaults are provided for the sake of extra documentation.
-    # grpc:
-    #   port: 8082
-    # initialDelaySeconds: 10
-    # periodSeconds: 10
-  # readinessProbe: {}
-    ## NOTE: Currently not configurable, defaults are provided for the sake of extra documentation.
-    # grpc:
-    #   port: 8082
-    # initialDelaySeconds: 5
-    # periodSeconds: 10
-    # failureThreshold: 10
+  
+  livenessProbe:
+    grpc:
+      port: 8082
+    initialDelaySeconds: 10
+    periodSeconds: 10
```

Just realized this is the default so we could just drop this (and from the defaults in the table as well)
```suggestion
    # periodSeconds: 10
```